models/models.js

```diff
@@ -33,7 +33,21 @@ exports.getArticleIdInfo = (article_id) => {
 exports.getComments = (article_id) => {
     return db.query('SELECT comment_id,votes,created_at,author,body FROM comments WHERE article_id=$1 ORDER BY created_at DESC;',[article_id])
     .then(({rows}) => {
-            return (rows);
+        return (rows);
     });
 };
 
+//8
+exports.updateArticleVotes = (article_id, votes) => {
+    if(Object.keys(votes).length!==1){
+        return Promise.reject({status:400, message: 'Missing information'})
```

Comment:
Great to get the invalid inputs out of the way before even trying to query the database 👍